Sources/TiqrCoreObjC/Classes/ScanOverlayView.m

```diff
@@ -80,7 +80,7 @@ - (void)drawRect:(CGRect)rect {
 	CGFloat white[4] = { 1.0f, 1.0f, 1.0f, 1.0f };
 	CGContextSetStrokeColor(context, white);
 	CGContextSetFillColor(context, white);
-	[self drawRect:self.cropRect inContext:context];
+//	[self drawRect:self.cropRect inContext:context];
```

Comment:
Commented Out Code Without Comment (cocwoc)

Reply:
I resolved this

Reply:
That's not really a fix. :-) Reading the code: this disables the white square on the screen for the user to put the QR-code in.

Reply:
Thanks for elaborating on that, the current QR screen doens't need the borde so I wil remove the code